spec/integration/v1/leads_spec.rb

```diff
@@ -0,0 +1,90 @@
+require 'swagger_helper'
+require 'rails_helper'
+#require 'vcr_helper'
+require 'spec_helper'
+
+RSpec.describe 'api/v1/leads', type: :request do
+  before do
+    allow(Rails.application.config).to receive(:consider_all_requests_local) { false }
+  end
+
+  before(:all) do
+    # needed for cookie check
+    contact = create_contact
+    @headers = set_cookie
+    @token_header = create_token_header
+    #@lead = FactoryBot.create(:api_lead, { salesforce_id: '0062F00000BG056QAD', contact_id: '0032F00000cfZQhQAM', school_id: '0012F00000iPxe9QAC' })
```

Comment:
remove?

Reply:
Forgot I didn't complete the integration test last week. Completed test with cassettes and updated swagger docs added.